domains/arnaz.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Arnaz's personal portfolio page",
+    "repo": "https://github.com/arnazmursidp/portofolio",
+    "owner": {
+        "username": "arnazmursidp",
+        "email": "arnazmursid@gmail.com"
+    },
+    "record": {
+        "CNAME": "arnazmursidp.github.io/portofolio"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/